composer/datasets/in_context_learning_evaluation.py

```diff
@@ -1343,105 +1576,159 @@ def get_icl_task_dataloader(
         max_seq_len: int,
         pad_tok_id: int,
         num_fewshot: int,
-        prompt_string: str,  # e.g. 'translate english to french:'
-        example_delimiter: str,  # e.g. '\n'
-        continuation_delimiter: str = '',
+        prompt_string: str = '',  # e.g. 'translate english to french:'
+        example_delimiter: str = '\n',  # e.g. '\n'
+        continuation_delimiter: str = ' ',
```

Comment:
I'm not sure the default here should be a space, as it might be unexpected for users doing their own evals.

Reply:
Agreed, but the change here is meant to mirror the default that [is set in llm-foundry](https://github.com/mosaicml/llm-foundry/blob/main/llmfoundry/utils/builders.py#L478). 
Those defaults will be inserted in for both `train.py` and `eval.py`